experimenter/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -135,6 +135,11 @@ select.form-control.is-warning {
   height: 18px;
 }
 
+button:disabled {
+  background-color: #6c757d !important;
+  border-color: #6c757d !important;
+}
```

Comment:
What is the selector that this is overriding?

Reply:
The changes in this file are no longer necessaary :)